app/src/main/java/nextstep/jwp/framework/http/HttpHeaders.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.jwp.framework.http;
+
+import static nextstep.jwp.framework.http.HttpRequest.LINE_DELIMITER;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HttpHeaders {
+
+    public static final String HEADER_DELIMITER = ":";
+    public static final String SPACE = " ";
+
+    private final Map<String, String> headers;
+
+    public HttpHeaders(final String lines) {
+        this.headers = convert(lines);
+    }
+
+    public HttpHeaders(final Map<String, String> headers) {
+        this.headers = headers;
+    }
```

Comment:
```suggestion
    public HttpHeaders(final String lines) {
        this(convert(lines));
    }

    public HttpHeaders(final Map<String, String> headers) {
        this.headers = headers;
    }
```
주생성자가 안 쓰이고 있네요! 부생성자를 주생성자를 이용하도록 만들어보는 건 어떨까요?
물론 이러면 `convert()` 메서드를 정적 메서드로 변경해야해요.